Home.md

```diff
@@ -1,40 +1,51 @@
-## Introduction
-[d:swarm](http://dswarm.org) is a **data management platform** that is based on the concept of [community sharing](http://en.wikipedia.org/wiki/Sharing). It is used for the lossless transformation of data from heterogeneous sources into a flexible ([elastic](http://en.wikipedia.org/wiki/Elasticity_%28data_store%29)), data model that can be explicitly made available as [(Linked) Open Data](http://en.wikipedia.org/wiki/Linked_data) (LOD).
+[d:swarm](http://dswarm.org) is a **data management platform** that can be used for the lossless transformation of data from heterogeneous sources into a flexible ([elastic](http://en.wikipedia.org/wiki/Elasticity_%28data_store%29)), data model. This data model can serve as a single source for providing  [Linked Open Data](http://en.wikipedia.org/wiki/Linked_data) (LOD).
 
 d:swarm is a **middle ware solution**. It forms the basis of all data management processes in a library or any other (cultural) institution dedicated to the handling of data and metadata.  Structurally, d:swarm goes in between **existing data management systems** (e.g. Integrated Library Systems) and **existing front end applications** (e.g. the library catalogue or discovery system).
 
-d:swarm (mainly) takes care of the **improvement** of the data quality facilitating the de-duplication, merging and [FRBRization](http://en.wikipedia.org/wiki/Functional_Requirements_for_Bibliographic_Records) of bibliographic data, their semantic enrichment and publication as LOD.
+[[http://www.dswarm.org/wp-content/uploads/2015/04/dswarm-demo_2015-04-14.png]]
 
-d:swarm channels all **institutional data flows** and is able to create a highly customized master record for every individual resource as a [Single Version of the Truth](http://en.wikipedia.org/wiki/Single_version_of_the_truth) (SVOT).
+Finally, d:swarm is an **ETL tool with a GUI for non-programmers**. Librarians do not need to write scripts, but can create complex transformations by Drag and Drop from a functions library and configuring them in the d:swarm BackOffice. Following the concept of [community sharing](http://en.wikipedia.org/wiki/Sharing), transformations, mappings and almost any other artifact in d:swarm that could be helpful to others is designed for **reuse**.

```

Comment:
"non-programmers" is negative, why not "... ETL tool with a GUI intended for knowledge workers, e.g., system librarians"